src/include/baselib/jni/JavaBridge.h

```diff
@@ -254,10 +254,30 @@ namespace bl
 
                 const callback_t& callback = *reinterpret_cast< callback_t* >( callbackAddress );
 
-                callback( inBuffer, outBuffer );
+
+                std::string exceptionText;
+                try
+                {
+                    callback( inBuffer, outBuffer );
+                }
+                catch( std::exception& e )
+                {
+                    exceptionText = e.what();
```

Comment:
is that adequate translation? most of the useful exception information will be lost, no?